include/alpaka/core/Cuda.hpp

```diff
@@ -177,13 +181,15 @@ namespace alpaka
                 if(error != CUDA_SUCCESS)
                 {
                     std::string const sError(std::to_string(file) + "(" + std::to_string(line) + ") '" + std::to_string(cmd) + "' returned error: '" + std::to_string(error) + "' (possibly from a previous CUDA call)!");
+#if ALPAKA_DEBUG != ALPAKA_DEBUG_DISABLED
```

Comment:
In all other places we currently use `ALPAKA_DEBUG >= ALPAKA_DEBUG_MINIMAL`. I would prefer this for consistency and because it makes it easier to increase the level of this debug message later.